openqa-investigate

```diff
@@ -49,9 +50,18 @@ clone() {
 
     name="$(echo "$clone_job_data" | runjq -r '.job.test'):investigate$name_suffix" || return $?
     base_prio=$(echo "$clone_job_data" | runjq -r '.job.priority') || return $?
+    worker_class=$(echo "$vars_json" | runjq -r '.WORKER_CLASS') || return $?
+    used_worker=$(echo "$clone_job_data" | runjq -r '.job.settings.WORKER_CLASS') || return $?
     clone_settings=("TEST+=:investigate$name_suffix" '_TRIGGER_JOB_DONE_HOOK=1' '_GROUP_ID=0' 'BUILD=')
+    mapfile -td ',' workers < <(echo "${worker_class}")
+    if [[  "${workers[@]}" =~ "$used_worker" ]]; then
+        if [[ $worker_class != null ]] && [[ $name_suffix =~ (last_good_test|last_good_build) ]]; then
+            clone_settings+=("WORKER_CLASS=${used_worker}")
+        fi
+    else
+        echoerr "WORKER_CLASS not found in vars.json"
```

Comment:
This "error" would end up in GRU minion logs on openQA executing this script but instead such message should go into the openQA comment informing openQA investigators about this caveat